Start of New Documentation

Aiidoneus opened the pull request:
This is the start of the new documentation while helping people understand what mods have been fixed by us and mod developers. By any means is it perfect but I think it does the job well. This closes #47

Aiidoneus commented:
I know bdew is working on adding fixes for most versions of his mod. I will wait and add them all at once once 1.12 is fixed an pushed out to the public See #47

Exaxxion commented on README.md:
I believe the last sentence of this paragraph should instead say "striving to keep" to maintain consistency with "trying to help" earlier in the sentence.

Aiidoneus replied:
Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

Exaxxion commented on README.md:
"developers adding patches" -> "developers add patches" to keep consistent with "as we find".

Probably don't need "to it" after "contribute" as well but that's a stylistic choice.

Aiidoneus replied:
Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

Exaxxion reviewed (commented):
Assuming chart is accurate, that part LGTM. Please see review comments for suggestions re: the ReadMe.

miozune commented:
AsieLib and TecTech from https://github.com/dogboy21/serializationisbad/pull/33 are not listed

Aiidoneus commented:
> AsieLib and TecTech from #33 are not listed

I re-add those soon.

Aiidoneus commented:
> AsieLib and TecTech from #33 are not listed

Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/fbaa9497d1e30fc83f632b7238b7678c56a56a20

miozune commented:
`TechTech` -> `TecTech`
Also [AsieLib](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L152-L160) and [TecTech](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L97-L105) are already patched

Aiidoneus commented:
Thanks for the catch. This is what I get for doing this at 3 in the morning. 
Fixed via https://github.com/dogboy21/serializationisbad/pull/48/commits/c8eb2014ddbd0023ce0258fb3011462d8c5bb630

Updated README in various places

dogboy21 commented on docs/mods.md:
```suggestion
|                                                    [mxTune](https://www.curseforge.com/minecraft/mc-mods/mxtune)                                                     |                                                                                                             Only affects Minecraft versions<br>`1.12` to `1.16.5`                                                                                                             |                                                  Nothing is fixed in the affected versions                                                   |                ✅ <br> [Project Patch](https://github.com/dogboy21/serializationisbad/blob/master/serializationisbad.json#L263-L271)                 |                                                                                          |                                                                                                                    |
```
